app/src/main/java/com/android/sample/ui/fridge/EditFridgeItemScreen.kt

```diff
@@ -114,7 +123,7 @@ fun EditComposable(
         Image(
             painter =
                 rememberAsyncImagePainter(
-                    model = ingredient.images[PRODUCT_FRONT_IMAGE_THUMBNAIL_URL]),
+                    model = ingredient.value[0].first.images[PRODUCT_FRONT_IMAGE_THUMBNAIL_URL]),
```

Comment:
I think we should up the quality to NORMAL, because it's the only image in this screen

Reply:
You're absolutely right. I plan to enhance the entire UI of the fridge ingredient edit screen next week. Since the primary focus of this PR is on the fridge UI and linking it with the `EditFridgeItemScreen`, I suggest addressing these improvements in the next update. Does that work for you?